ds3-sdk/src/main/java/com/spectralogic/ds3client/metadata/MetadataReceivedListenerImpl.java

```diff
@@ -0,0 +1,254 @@
+            final String[] ordinalArr = permission.split("_");
+            for (String ordinal : ordinalArr) {
+                switch (ordinal) {
+                    case "0":
+                        permissions.add(AclEntryPermission.READ_DATA);
+                        break;
+                    case "1":
+                        permissions.add(AclEntryPermission.WRITE_DATA);
+                        break;
+                    case "2":
+                        permissions.add(AclEntryPermission.APPEND_DATA);
+                        break;
+                    case "3":
+                        permissions.add(AclEntryPermission.READ_NAMED_ATTRS);
+                        break;
+                    case "4":
+                        permissions.add(AclEntryPermission.WRITE_NAMED_ATTRS);
+                        break;
+                    case "5":
+                        permissions.add(AclEntryPermission.EXECUTE);
+                        break;
+                    case "6":
+                        permissions.add(AclEntryPermission.DELETE_CHILD);
+                        break;
+                    case "7":
+                        permissions.add(AclEntryPermission.READ_ATTRIBUTES);
+                        break;
+                    case "8":
+                        permissions.add(AclEntryPermission.WRITE_ATTRIBUTES);
+                        break;
+                    case "9":
+                        permissions.add(AclEntryPermission.DELETE);
+                        break;
+                    case "10":
+                        permissions.add(AclEntryPermission.READ_ACL);
+                        break;
+                    case "11":
+                        permissions.add(AclEntryPermission.WRITE_ACL);
+                        break;
+                    case "12":
+                        permissions.add(AclEntryPermission.WRITE_OWNER);
+                        break;
+                    case "13":
+                        permissions.add(AclEntryPermission.SYNCHRONIZE);
+                        break;
+                }
+            }
+            builderWindow.setPrincipal(bRiceUser);
+            builderWindow.setPermissions(permissions);
+            builderWindow.setType(AclEntryType.ALLOW);
+            final AclEntry aclEntry = builderWindow.build();
+            aclEntries.add(aclEntry);
+        } catch (final Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    //set the last modified time saved on server
+    private void setModifiedTime(final String filePath, final Long modifiedTime) {
+        try {
+            final Path file = Paths.get(filePath);
+            final BasicFileAttributes attr =
+                    Files.readAttributes(file, BasicFileAttributes.class);
+            final FileTime ft = FileTime.fromMillis(modifiedTime);
+            Files.setLastModifiedTime(file, ft);
+        } catch (final Exception e) {
+            e.printStackTrace();
```

Comment:
Exceptions should not be handled in this way. At a minimum it should be logged with a slf4j logger, or wrapped in a runtime exception and re-thrown.